lib/std/os/windows.zig

```diff
@@ -640,6 +642,7 @@ pub fn CreateSymbolicLink(
         error.NotDir => unreachable,
         error.WouldBlock => unreachable,
         error.PipeBusy => unreachable,
+        error.AntivirusInterference => unreachable,
```

Comment:
Is this unreachable? Can't it be reached if `NtCreateFile` returns `VIRUS_INFECTED` or `VIRUS_DELETED`? If it can't, maybe could there be a comment with a link to a some microsoft doc page where it says that?

Reply:
i dont understand the underlying api but i think this isn't possible for the `FILE_CREATE` disposition. it would hit the path already exists error instead in every case, unless anti-virus is possible to block this call, but i'm not certain.

i had to stare at this code for a minute to reach that conclusion, with what i already know, and that might not even be the right answer.

Reply:
This may be a good time to transition to the new policy of using `error.Unexpected` instead of `unreachable` for these APIs.